crates/handlers/src/graphql/mod.rs

```diff
@@ -501,3 +502,13 @@ where
         session.map(Into::into).unwrap_or_default()
     }
 }
+
+/// A filter for dates, with a lower bound and an upper bound
+#[derive(InputObject, Default, Clone, Copy)]
+pub struct DateFilter {
+    /// The lower bound of the date range
```

Comment:
I think this is what gets generated as the GraphQL documentation right?

I think we should describe the format of the date here, since it's not obvious how a caller should use it.

I then wonder how much we care about timezones. Possibly not at all, given 90 days is far enough away that it probably doesn't matter whether it's a few hours off.

Reply:
The scalar used is a custom one, which is serialises as a string, and has the following (autogenerated) documentation on it:

```graphql
"""
Implement the DateTime<Utc> scalar

The input/output is a string in RFC3339 format.
"""
scalar DateTime
```

so that's probably fine?